pkg/protecode/protecode.go

```diff
@@ -169,6 +169,9 @@ func (pc *Protecode) mapResponse(r io.ReadCloser, response interface{}) {
 func (pc *Protecode) sendAPIRequest(method string, url string, headers map[string][]string) (*io.ReadCloser, error) {
 
 	r, err := pc.client.SendRequest(method, url, nil, headers, nil)
+	if err != nil {
+		return nil, err
+	}
 
 	return &r.Body, err
```

Comment:
Could make it more clear by returning `&r.Body, nil` here.

Reply:
```suggestion
	return &r.Body, nil
```